### PR TITLE
Added redux-saga example to README and ExNavigationActions export

### PR DESCRIPTION
**Pull request by kpyorala**
* Added redux-saga example to README.md
* Fixed couple of errors from “Integrate with your existing Redux
store” examples
* Added export of ExNavigationActions to src/ExNavigation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -482,20 +494,48 @@ import {
...
+/* In your saga middleware code */
+import { takeEvery } from 'redux-saga'
+import { put, select } from 'redux-saga/effects'
+import { NavigationActions } from '@exponent/ex-navigation'
+import Router from './Router'
+import selectors from './selectors'
+
+function* goHome() {
+  const navigatorUID = yield select(selectors.getNavigatorUID)
+  yield put(NavigationActions.push(navigatorUID, Router.getRoute('home')))
```
Comment (chirag04) [inline README.md]: This is only going to confuse more people. One can simply call `push` on the navigator.

Comment (kpyorala) [inline README.md]: OK, can you write an example how to use navigator outside components. There wasn't any instructions and this was the way @brentvatne aided me to do it in slack and he asked to do a pull request  for this.

Comment (brentvatne) [inline README.md]: @chirag04 - yep, I asked @kpyorala for this, I think it is good to have some documentation around how to use ex-navigation outside of the navigation context. Thanks @kpyorala :)

Comment (chirag04) [inline README.md]: gotcha. @kpyorala didn't mean to discourage you. example of `how to use navigator outside componets` is def valuable tho. maybe redux-thunk? idk

**Comment by brentvatne** [comment]
Merged, thanks @kpyorala! I also went ahead and modified the example to make it more generic, so it doesn't require people understand redux-saga.

**Pull request closed without merging**